gslab_scons/size_warning.py

```diff
@@ -128,3 +184,30 @@ def create_size_dictionary(dirs):
 
     return size_dictionary
 
+def add_to_lfs(filepath, attrib_path = '../.gitattributes'):
+    '''
+    This function tracks existing files with git lfs by writing down 
+    the given file path to the git attribute file (the default path is '../.gitattributes')
+    '''
+    with open(attrib_path, "a") as f:
+        f.write('\n%s filter=lfs diff=lfs merge=lfs -text' % filepath)
```

Comment:
I think we want to have '%s filter=lfs diff=lfs merge=lfs -text\n'. The current git attributes has a blank line at the end. Writing this would leave a blank line in the middle, that would throw the error what splitting.